modules/apps/trash/trash-taglib/src/main/resources/META-INF/resources/undo/page.jsp

```diff
@@ -90,8 +90,8 @@ int trashedEntriesCount = GetterUtil.getInteger(request.getAttribute("liferay-tr
 		<aui:input name="redirect" type="hidden" value="<%= redirectURL %>" />
 		<aui:input name="restoreTrashEntryIds" type="hidden" value="<%= StringUtil.merge(restoreTrashEntryIds) %>" />
 
-		<aui:button cssClass="alert-link btn-link btn-sm btn-unstyled trash-undo-button" type="submit" value="undo" />
+		<aui:button cssClass="alert-link btn-link btn-sm btn-unstyled trash-undo-button" primary="false" type="submit" value="undo" />
```

Comment:
This looks weird... is `primary` a boolean attribute? If so, this should be `primary="<%= false %>"`

Reply:
According https://github.com/liferay/liferay-portal/blob/master/util-taglib/src/META-INF/liferay-aui.tld#L204 is a java.lang.Object